packages/flutter_tools/templates/create/.travis.yml.tmpl

```diff
@@ -0,0 +1,11 @@
+os:
+  - osx
+osx_image: xcode7.2
+sudo: false
+before_script:
+  - git clone https://github.com/flutter/flutter.git /tmp/flutter
```

Comment:
> 1. This is tracking `master` rather than `alpha`.  Really, the developer should decide on a specific git hash for flutter.git that they update in a controlled manner.  As written, the Travis for a customer will go red whenever we land a change in `master` that breaks their app.

Agreed. Though, this is only a template. After the initial `flutter create`, the developer is in full (and sole) control of the repo and the branch. Should I modify this to default to `alpha` instead?

> 2. Putting the flutter repo in `/tmp/flutter` isn't the pattern we've been recommending.  Instead, we've been recommending that people put flutter in a peer directory to their project.  For example, they'd have a path dependency to `../flutter/packages/flutter`.

Since putting the Flutter repo next to the project is only a recommendation (one our users may choose to ignore), I decided to setup the test harness so that it downloads the Flutter repo at a known location and unconditionally rewrite the path dependency.